### PR TITLE
Implemented chat on the session view

### PR DESCRIPTION
**Pull request by Laugslander**

### CODE REVIEW

File: Soccar [UI]/src/nl/socnet/message/ChatMessage.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.CHAT_MESSAGE_ID, handler = ChatMessageHandler.class)
+public final class ChatMessage extends Message {
+
+    private final String username;
+    private final Privilege privilege;
+    private final String message;
+
+    public ChatMessage(String username, Privilege privilege, String message) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/ChatMessage.java:20]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [UI]/src/nl/socnet/message/ChatMessage.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        this.username = username;
+        this.privilege = privilege;
+        this.message = message;
+    }
+
+    public ChatMessage(String message) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/socnet/message/ChatMessage.java:26]: MINOR Document this public constructor. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

File: Soccar [UI]/src/nl/soccar/ui/fx/controller/SessionViewFXMLController.java [hunk 3]
```diff
@@ -121,12 +151,45 @@ private void leaveRoom() {
...
+        if (message.trim().isEmpty() || message.length() >= 75) {
+            return;
+        }
+        
+        ClientController.getInstance().getCurrentConnection().send(new ChatMessage(message));
+        
+        txtChat.clear();
+    }
+    
+    public void addChatMessage(String username, Privilege privilege, String message) {
```
Comment (ghost) [inline Soccar [UI]/src/nl/soccar/ui/fx/controller/SessionViewFXMLController.java:177]: MINOR Document this public method. [rule](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)

**Comment by ghost** [comment]
SonarQube analysis reported 3 issues
* MINOR 3 minor

Watch the comments in this conversation to review them.

**Comment by BeeHiveJava** [comment]
retest this please

**Review by Ranray — APPROVED** [review]

**Pull request merged**